database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
-    "level_3_lighting_90_1_2022_prm": "todo",
-    "level_3_lighting_IECC_2006": "todo",
-    "level_3_lighting_IECC_2009": "todo",
-    "level_3_lighting_IECC_2012": "todo",
-    "level_3_lighting_IECC_2015": "todo",
-    "level_3_lighting_IECC_2018": "todo",
-    "level_3_lighting_IECC_2021": "todo",
-    "level_3_ventilation_62_1_1999": "todo",
-    "level_3_ventilation_62_1_2004": "todo",
-    "level_3_ventilation_62_1_2007": "todo",
-    "level_3_ventilation_62_1_2010": "todo",
-    "level_3_ventilation_62_1_2013": "todo",
-    "level_3_ventilation_62_1_2016": "todo",
-    "level_3_ventilation_62_1_2019": "todo",
-    "level_3_ventilation_62_1_2022": "todo",
-    "support_constructions": "todo",
-    "support_electric_equipment_space_type_name_tags": "todo",
-    "support_lighting_space_type_name_tags": "todo",
-    "support_lighting_technologies": "todo",
-    "support_materials": "todo",
-    "support_performance_curves": "todo",
-    "support_schedules": "todo",
-    "support_standard_templates": "todo",
-    "support_ventilation_space_type_name_tags": "todo",
-    "system_requirements_air_economizer_90_1": "todo",
-    "system_requirements_air_economizer_90_1_prm": "todo",
-    "system_requirements_air_economizer_IECC": "todo",
-    "system_requirements_energy_recovery_90_1": "todo",
-    "system_requirements_energy_recovery_90_1_prm": "todo",
-    "system_requirements_energy_recovery_IECC": "todo"
+    "envelope_requirements_90_1": "Envelope requirements for ASHRAE Standard 90.1",
+    "envelope_requirements_90_1_prm": "Envelope requirements for ASHRAE Standard 90.1 Appendix G",
+    "envelope_requirements_IECC": "Envelope requirements for the IECC",
+    "exterior_lighting_90_1": "Exterior lighting requirements for ASHRAE Standard 90.1",
+    "exterior_lighting_90_1_prm": "Exterior lighting requirements for ASHRAE Standard 90.1 Appendix G",
+    "exterior_lighting_IECC": "Exterior lighting requirements for the IECC",
+    "hvac_minimum_requirements_boilers_90_1": "Minimum efficiency requirements boilers for ASHRAE Standard 90.1",
+    "hvac_minimum_requirements_boilers_90_1_prm": "Minimum efficiency requirements boilers for ASHRAE Standard 90.1 Appendix G",
+    "hvac_minimum_requirements_boilers_IECC": "Minimum efficiency requirements boilers for the IECC",
+    "hvac_minimum_requirements_chillers_90_1": "Minimum efficiency requirements chillers for ASHRAE Standard 90.1",
+    "hvac_minimum_requirements_chillers_90_1_prm": "Minimum efficiency requirements chillers for ASHRAE Standard 90.1 Appendix G",
+    "hvac_minimum_requirements_chillers_IECC": "Minimum efficiency requirements chillers for the IECC",
+    "hvac_minimum_requirements_computer_room_air_conditioners_90_1": "Minimum efficiency requirements computer room air-conditioners for ASHRAE Standard 90.1",
+    "hvac_minimum_requirements_computer_room_air_conditioners_IECC": "Minimum efficiency requirements computer room air-conditioners for the IECC",
```

Comment:
Should add "for" before equipment for consistency? 

e.g., "for boilers", "for chillers", etc..

Reply:
Yes!